generatebundlefile/scripts/common.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
```

Comment:
Is there any reason you can't just use the one that already exists in the $ROOT/scripts folder?

Reply:
I didn't realize we had the same common script already copied into the $ROOT. Can just that.